src/main/java/Lesson5/Human.java

```diff
@@ -0,0 +1,15 @@
+package Lesson5;
+
+public class Human
+{
+    public String name;
+    public String surname;
+    public String gender;
+    public int age;
+    public Human(String Name,String Surname,String Gender,int Age){
```

Comment:
in java all variable names starting from lowercase

Reply:
ok